site/app/templates/polls/ViewPoll.twig

```diff
@@ -95,6 +107,50 @@
 </div>
 
 <script>
+
```

Comment:
Could we move this to js file so that it can get linted? if js file does not exist lets make one.